plugins/woocommerce-blocks/bin/webpack-entries.js

```diff
@@ -19,6 +19,9 @@ const blocks = {
 	'add-to-cart-form': {
 		customDir: 'product-elements/add-to-cart-form',
 	},
+	'add-to-cart-with-options': {
+		customDir: 'add-to-cart-with-options',
```

Comment:
Set as experimental here. This is a PR blocker 🚫

Reply:
Added in the commit [53422cf](https://github.com/woocommerce/woocommerce/pull/53215/commits/53422cfcb5588296187504580915e644b60e3c94)